HW3/task1.py

```diff
@@ -0,0 +1,13 @@
+# Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами. 
+# В результирующем списке не должно быть дубликатов.
+
+my_list = [1, 1, "Hi", "Hi", 5, 6, 0.5, 0.5]
+
+ENTERS = 2
+result_list = []
+
+for item in set(my_list):
+    if my_list.count(item) == ENTERS:
```

Comment:
А почему вы сравниваете с 2, когда количество повторений в условии не указано
```suggestion
    if my_list.count(item) > 1:
```

Reply:
Может тогда уж так: 
```suggestion
    if my_list.count(item) >= ENTERS:
```
Чтобы не было "магических чисел"?